src/metpy/calc/thermo.py

```diff
@@ -1636,7 +1636,7 @@ def saturation_equivalent_potential_temperature(pressure, temperature):
     e = saturation_vapor_pressure(temperature).to('hPa').magnitude
     r = saturation_mixing_ratio(pressure, temperature).magnitude
 
-    th_l = t * (1000 / (p - e)) ** mpconsts.kappa
+    th_l = t * (1000 / (p - e)) ** mpconsts.kappa.magnitude
```

Comment:
`kappa` is available in `metpy.constants.nounit`

Reply:
Good catch. That commit is from 10/6/2020, so I think it actually predates us having that available. 😆